### PR TITLE
[IMP] mail: add search view in message subtype

### PR DESCRIPTION
**Pull request by mrsr-odoo**
This PR add a search view to the message subtype model.
-Add the 'model' and 'description' fields in the search view. 
-Add 'default' filter.

Task-3459860

### CODE REVIEW

File: addons/mail/views/mail_message_subtype_views.xml [hunk 1]
```diff
@@ -43,6 +43,19 @@
             </field>
         </record>
 
+        <record model="ir.ui.view" id="view_mail_message_subtype_search">
```
Comment (tde-banana-odoo) [inline addons/mail/views/mail_message_subtype_views.xml:46]:
```suggestion
        <record model="ir.ui.view" id="mail_message_subtype_view_search">
```

File: addons/mail/views/mail_message_subtype_views.xml [hunk 2]
```diff
@@ -43,6 +43,19 @@
...
+            <field name="name">mail.message.subtype.search</field>
```
Comment (tde-banana-odoo) [inline addons/mail/views/mail_message_subtype_views.xml:47]:
```suggestion
            <field name="name">mail.message.subtype.view.search</field>
```

**Review by tde-banana-odoo — COMMENTED** [review]
Additional :socks:

**Comment by awa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**